modules/netweaver/main.tf

```diff
@@ -31,6 +31,10 @@ resource "google_compute_disk" "gcp_nw_pd_0" {
   zone    = var.zone
   count   = var.usr_sap_size > 0 ? 1 : 0
   size    = var.usr_sap_size
+
+  disk_encryption_key {
```

Comment:
Can we make this a conditional block so it's excluded entirely if key isn't specified?

Reply:
I am using this to omit these blocks if no CMEK key was provided. Is this the best way of doing it?
```
  dynamic "disk_encryption_key" {
    for_each = var.pd_kms_key != null ? [""] : []
    content {
      kms_key_self_link = var.pd_kms_key
    }
  }
```